nginx_proxy/start_nginx.sh

```diff
@@ -20,6 +20,7 @@ CRT_FILE=${CERT_DIR}/lb.crt
 
 ENDPOINTS_SERVICE_NAME=''
 ENDPOINTS_SERVICE_VERSION=''
+ENDPOINTS_ROLLOUT_STRATEGY=''
```

Comment:
Please default this to fixed to match the comment on line 56.

Reply:
Done